docs/getting-started/in-cluster-operator.md

```diff
@@ -0,0 +1,79 @@
+# K8sGPT Operator
+
+## Prerequisites
+
+- To begin you will require a Kubernetes cluster available and `KUBECONFIG` set.
+- You will also need to install helm v3. [See the Helm documentation for more information](https://helm.sh/docs/intro/install/).
+
+## Operator Installation
+
+To install the operator, run the following command:
+
+```bash
+helm repo add k8sgpt https://charts.k8sgpt.ai/
+helm install release k8sgpt/k8sgpt-operator
+```
+
+This will install the Operator into the cluster, which will await a `K8sGPT` resource before anything happens.
+
+## Deploying an OpenAI secret
+
+Whilst there are multiple backends supported ( OpenAI, Azure OpenAI and Local ), in this example we'll use OpenAI.
+
+This means you will need to install your token as a secret into the cluster.
+
+For example:
+
+```bash
+kubectl create secret generic k8sgpt-sample-secret --from-literal=openai-api-key=$OPENAI_TOKEN -n default
+```
+
+## Deploying a K8sGPT resource
+
+To deploy a K8sGPT resource, you will need to create a YAML file with the following contents:
+
+```yaml
+kubectl apply -f - << EOF
+apiVersion: core.k8sgpt.ai/v1alpha1
+kind: K8sGPT
+metadata:
+  name: k8sgpt-sample
+spec:
+  namespace: default
+  model: gpt-3.5-turbo
+  backend: openai
+  noCache: false
+  version: v0.2.7
+  enableAI: true
+  secret:
+    name: k8sgpt-sample-secret
+    key: openai-api-key
+EOF
+```
+
+As you can see above many of the settings are configurable and `AI` can either be turned on or off easily.
```

Comment:
Could you please replace this line with the different configuration that people would change i.e. provide a little more detail/explanation e.g. why would someone turn "AI" off i.e. set it to false?